wolfcrypt/test/test.c

```diff
@@ -19613,6 +19613,7 @@ int memcb_test(void)
 exit_memcb:
 
     /* restore memory callbacks */
+    wolfSSL_ResetAllocators();
```

Comment:
Is this the right solution?  That is, wouldn't it be easier for users if SetAllocators called ResetAllocators?  Or SetAllocators allowed NULLs?  If this bit us, I think we want to make it easier for users.